modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc

```diff
@@ -0,0 +1,78 @@
+= Amazon S3 Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install Amazon S3
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. Search for the Amazon S3 connector, click on the connector, and click *Add to Project*.
+. Follow the prompts.
+
+== Configure Amazon S3
+
+To configure this connector:
+
+. Drag and drop a connector operation into the Studio canvas.
+. To create a global element for the connector, click the *Global Elements* tab at the base of the canvas, and then click *Create*.
+. In the *Choose Global Type* window, expand *Connector Configuration*, and click *Amazon S3 Configuration*.
+. Click *OK*.
+. Select *Amazon S3 Configuration (Configuration)*, and click *Edit*.
+. Specify the global element properties:
++
+image::amazon/amazon-s3-01.png[Global Element Properties]
++
+[%header%autowidth.spread]
+|===
+|Parameter |Description
+|Name |Name used to reference the connector instance.
+|Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
```

Comment:
Need to say where to get these (they come from Amazon), but at least say that, and also check out Amazon to see how much work it is to find those values.

Reply:
AWS is Amazon Web Services - I don't see the need to further qualify that this is AWS is an Amazon offering.